crates/model/src/order.rs

```diff
@@ -915,9 +906,6 @@ impl OrderClass {
 #[derive(Eq, PartialEq, Clone, Copy, Debug, Default, Hash, Deserialize, Serialize)]
 #[serde(rename_all = "camelCase")]
 pub struct LimitOrderClass {
-    #[serde_as(as = "Option<DecimalU256>")]
-    pub surplus_fee: Option<U256>,
-    pub surplus_fee_timestamp: Option<DateTime<Utc>>,
     #[serde_as(as = "Option<DecimalU256>")]
     pub executed_surplus_fee: Option<U256>,
```

Comment:
Not related to this PR, but would LOVE to see this property go as well (I think it requires different `Order` types for different situations - for example this is only needed by the `orderbook` API on `GET api/v1/orders/${uid}` to be able to display executed fee amounts, but not in the order used for the `GET api/v1/auction`).